ykman-gui/qml/PivSetManagementKeyView.qml

```diff
@@ -205,14 +182,47 @@ ColumnLayout {
                     ToolTip.delay: 1000
                     ToolTip.visible: hovered
                     ToolTip.text: qsTr("Management key must be exactly 48 hexadecimal digits.")
-                    visible: hasNewManagementKeyInput
                 }
                 CustomButton {
                     id: randomManagementKeyBtn
                     text: qsTr("Generate")
                     Layout.alignment: Qt.AlignHCenter | Qt.AlignVCenter
                     onClicked: generateManagementKey()
-                    visible: hasNewManagementKeyInput
+                }
+
+                Label {}
+                CheckBox {
+                    id: storeManagementKeyCheckbox
+                    checked: true
+                    text: qsTr("Store on YubiKey, protected by PIN")
+                    font.pixelSize: constants.h3
+                    Material.foreground: yubicoBlue
+                }
+                Label {}
+
+                Label {
+                    text: qsTr("PIN:")
+                    font.pixelSize: constants.h3
+                    color: yubicoBlue
+                    Layout.alignment: Qt.AlignRight | Qt.AlignVCenter
+                    visible: hasPinInput
+                }
+                TextField {
+                    id: pin
+                    Layout.fillWidth: true
+                    echoMode: enabled ? TextInput.Password : TextInput.Normal
+                    selectByMouse: true
+                    selectionColor: yubicoGreen
+                    visible: hasPinInput
```

Comment:
I'd prefer if toggling the "Use PIN" checkbox enables/disables the input rather than making it invisible (it makes the UI less "jumpy".)